### ISSUE TITLE
Regenerate answer if the question is edited. Introduce regenerate answer button like in ChatGPT.

### ISSUE THREAD

**Issue by BoQsc** [ztjhz/BetterChatGPT#179]

**Comment by ztjhz** [comment]
If you click "save and submit" after editing a message, it will regenerate the responses. Alternatively, you can click on the refresh icon at the bottom right of the answer

![refresh icon](https://user-images.githubusercontent.com/59118459/229288555-c7e7b846-6991-42cf-b7b9-007ced890cb5.jpg)

![Save and submit](https://user-images.githubusercontent.com/59118459/229288598-0036b8c2-179b-41fc-b93d-391b0541408a.jpg)

**Comment by BoQsc** [comment]
@ztjhz regeneration only available to the latest answer.
![image](https://user-images.githubusercontent.com/21064622/229291813-8d607c08-7995-4076-8ac4-b6f8c8039b03.png)

**Comment by ztjhz** [comment]
Yes, it is only available for the last message if it is an assistant. We can add regeneration for all assistant messages.
